tests/smoke/kotlin/src/test/kotlin/ProcStat.kt

```
@@ -43,56 +42,96 @@ class ProcStat {
+
+                    "upload-status" -> {
+                        val filename = report.content.filename
+                        val tguid = report.content.tguid
+                        val offset = report.content.offset
+                        val size = report.content.size
+                        log.info("Upload Status - Filename: $filename, TGUID: $tguid, Offset: $offset, Size: $size")
+                    }
+
+                    "blob-file-copy" -> {
+                        val sourceUrl = report.content.fileSourceBlobUrl
+                        val destinationUrl = report.content.fileDestinationBlobUrl
+                        val destinationName = report.content.destinationName
+                        assertNotNull(sourceUrl, "Blob source URL is missing for $schemaName")
+                        assertNotNull(destinationUrl, "Blob destination URL is missing for $schemaName")
+                        assertNotNull(destinationName, "Destination name is missing for $schemaName")
+                        log.info("Blob File Copy - Source URL: $sourceUrl, Destination URL: $destinationUrl")
+
+                        val expectedDestinations = config.copyConfig.targets
+
+                        val actualDestination = expectedDestinations.any { expectedDest ->
+                            destinationName?.trim()?.equals(expectedDest.trim(), ignoreCase = true) ?: false
+                        }
+
+                        assertTrue(
+                            actualDestination,
+                            "None of the expected destinations ('${expectedDestinations.joinToString(", ")}') were found in the response. Found: $destinationName"
+                        )
+                    }
+
+                    else -> {
+                        log.info("Unknown schema name: $schemaName")
+                    }
+                }
+            }
+
+        } catch (e: Exception) {
+            log.error("Test failed due to exception: ${e.message}", e)

```

This catch is probably unnecessary.  Just let the test fail instead of catching the exception